test/models/TaxGroupTypeSpec.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import utils.SpecBase
+import models.TaxGroupType
+import play.api.libs.json.{JsValue, Json, JsString, JsSuccess}
+
+class TaxGroupTypeSpec extends SpecBase {
+
+  "TaxGroupType" must {
```

Comment:
Please follow the comments that has been given for CDSAccountStatusSpec (apart from the file rename)